Src/Newtonsoft.Json/Converters/XmlNodeConverter.cs

```diff
@@ -986,7 +986,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
 
             IXmlNode node = WrapXml(value);
 
-            XmlNamespaceManager manager = new XmlNamespaceManager(new NameTable());
+            XmlNamespaceManager manager = new XmlNamespaceManager(new System.Xml.NameTable());
```

Comment:
Hmmm, this might be annoying for people when updating their NuGet packages. To avoid this collision maybe it should be JsonNameTable and DefaultJsonNameTable?

Reply:
Fixed.